tendermint/src/block/commit.rs

```diff
@@ -4,11 +4,12 @@ use crate::{block, Vote};
 use serde::{Deserialize, Serialize};
 use std::{ops::Deref, slice};
 
-/// Last commit to a particular blockchain: +2/3 precommit signatures.
+/// Commit contains the evidence that a block was committed by a set of validators.
```

Comment:
```suggestion
/// Commit contains the justification (ie. a set of signatures) that a block was committed by a set of validators.
```

Best not to use `evidence` since it has another meaning within blocks.